### PR TITLE
Updating package to reflect new release.

### PR DESCRIPTION
**Pull request by mhearne-usgs**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
-{% set version = "v0.1.3" %}
+{% set version = "0.1.3" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @mhearne-usgs version number cannot start with a letter or the solver may not show this version as the latest one.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 package:
   name: gmprocess
   version: {{ version }}
 
 source:
-  url: https://github.com/usgs/groundmotion-processing/archive/{{ version }}.tar.gz
+  url: https://github.com/usgs/groundmotion-processing/archive/v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: It seems that the `v` in the GitHub tarball comes and goes on new releases, if you can make that URL stable the bot will get the releases automatically for you. (Or perhaps use a PyPI release.)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
   sha256: 275a470a20f9c883049d54593016b2151c9251212f46b2325407c7042f5e33f2
 
 build:
-  number: 2
+  number: 0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Remember to reset the build number to `0` when issuing a new release.

File: recipe/meta.yaml (hunk 2 above)
Comment (mhearne-usgs) [reply recipe/meta.yaml]: I screwed up with the last version by leaving off the v. I'll reset the build number and turn off windows builds for now until I figure out how that should work.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by mhearne-usgs** [comment]
@ocefpaf Why are there two Travis builds? Also, I heard about conda-forge switching to Azure, is that what the last two builds are?

**Comment by ocefpaf** [comment]
> @ocefpaf Why are there two Travis builds?

I believe that the hooks travis-ci.com and travis.org may be at play here, not sure. It is safe to ignore though.

> Also, I heard about conda-forge switching to Azure, is that what the last two builds are?

Nope. The azure are the "space ship" icons below.

**Pull request merged**